patroni/dcs/__init__.py

```diff
@@ -956,36 +957,37 @@ def __permanent_logical_slots(self) -> Dict[str, Any]:
         """Dictionary of permanent ``logical`` replication slots."""
         return {name: value for name, value in self.__permanent_slots.items() if self.is_logical_slot(value)}
 
-    def get_replication_slots(self, my_name: str, role: str, nofailover: bool, major_version: int, *,
-                              show_error: bool = False) -> Dict[str, Dict[str, Any]]:
+    def get_replication_slots(self, postgresql: 'Postgresql', member: Tags, *,
+                              role: Optional[str] = None, show_error: bool = False) -> Dict[str, Dict[str, Any]]:
         """Lookup configured slot names in the DCS, report issues found and merge with permanent slots.
 
         Will log an error if:
 
             * Any logical slots are disabled, due to version compatibility, and *show_error* is ``True``.
 
-        :param my_name: name of this node.
-        :param role: role of this node.
-        :param nofailover: ``True`` if this node is tagged to not be a failover candidate.
-        :param major_version: postgresql major version.
+        :param postgresql: reference no :class:`Postgresql` object.
```

Comment:
%s/no/to

Reply:
@zhjwpku you can leave suggestions directly here, like below:
```suggestion
        :param postgresql: reference to :class:`Postgresql` object.
```

Reply:
> @zhjwpku you can leave suggestions directly here, like below:

Sorry for the inconvenience, I don't know how to do that, I will figure it out later.